app/templates/search.hbs

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
you solved the little footer line problem. good job! how did you do it?

Reply:
yes. I added a custom class and changed the font styles.